spec/mock/json_to_html_mock.rb

```diff
@@ -43,6 +43,7 @@
 EntryReferenceLinkJson = '{ "uid":"06e34a7  5e4 e549d", "_version":1, "attrs":{ }, "children":[{"uid":"7626ea98e0e95d602210","type":"reference","attrs":{"target":"_self","href":"/copy-of-entry-final-02","display-type":"link","entry-uid":"entry_uid_2","content-type-uid":"embeddedrte","locale":"en-us","type":"entry","class-name":"embedded-entry"},"children":[{"text":"/copy-of-entry-final-02"}]}],"type":"doc"}'
 EntryReferenceInlineJson = '{ "uid":"06e34a7  5e4 e549d", "_version":1, "attrs":{ }, "children":[{"uid":"506 4878f3f46 s21f0cbc aff","type":"reference","attrs":{"display-type":"inline","entry-uid":"entry_uid_3","content-type-uid":"embeddedrte","locale":"en-us","type":"entry","class-name":"embedded-entry"},"children":[{"text":""}]}],"type":"doc"}'
 HRJson = '{ "uid":"06e34a7  5e4 e549d", "_version":1, "attrs":{ }, "children":[{"uid":"f5a7b57 40a8a5c3 576828276b","type":"hr","children":[{"text":""}],"attrs":{ }}],"type":"doc"}'
+BRJson = '{ "uid":"06e34a7  5e4 e549d", "_version":1, "attrs":{ }, "children":[{"uid":"f5a7b57 40a8a5c3 576828276b","type":"br","children":[{"text":""}],"attrs":{ }}],"type":"doc"}'
```

Comment:
the br tag comes in the text not as type. similar to other **mark types** like **subscripts** and **bold**

Reply:
Done